paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/Spec.kt

```diff
@@ -86,4 +86,9 @@ sealed class SectionFieldSpec(val identifier: IdentifierSpec) {
         SectionFieldSpec(IdentifierSpec("country"))
 
     object IdealBank : SectionFieldSpec(IdentifierSpec("bank"))
+
+    data class GenericText(
+        override val identifier: IdentifierSpec,
```

Comment:
will this identifier always be `IdentifierSpec("genericText")`? if so we can avoid opening identifier as
```
data class GenericText(
        @StringRes val label: Int
    ) : SectionFieldSpec(IdentifierSpec("genericText"))
```

Reply:
This identifier might be line1 or line2 or state, depending on how it is defined by the creator of the spec.